contracts/evmscript/EVMScriptRegistry.sol

```diff
@@ -19,8 +19,11 @@ contract EVMScriptRegistry is IEVMScriptRegistry, EVMScriptRegistryConstants, Ar
     // WARN: Manager can censor all votes and the like happening in an org
```

Comment:
Note that upgrading this app for existing organizations will be a PITA; we've [pinned these apps to their current implementations](https://github.com/aragon/aragonOS/blob/dev/contracts/factory/EVMScriptRegistryFactory.sol#L23) and so these orgs will have to reset their default `EVMScriptRegistry` app in the kernel to get this change (along with uninstalling the old one completely, for the frontend to not look weird).